R/test_datasets.R

```diff
@@ -0,0 +1,633 @@
+#' Test whether specified dataset_id has the correct setup
```

Comment:
I've looked through much of the code for this file, but am not sure I'd pick up if there were something specific that isn't right, since I've never written the tests. The lines of code I've read are all logical and I assume this is mostly moved to this file from those you've deleted, not primarily written from scratch?